toolchain/check/testdata/interop/cpp/import.carbon

```diff
@@ -0,0 +1,120 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// INCLUDE-FILE: toolchain/testing/testdata/min_prelude/int.carbon
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/interop/cpp/import.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/interop/cpp/import.carbon
+
+// --- header.h
+
+auto foo_short(short x) -> void;
+auto foo_int(int x) -> void;
+
+// --- api.carbon
+
+library "[[@TEST_NAME]]";
+
+import Cpp library "header.h";
+
+alias FooShort = Cpp.foo_short;
+alias FooInt = Cpp.foo_int;
+
+// --- fail_todo_import_api.carbon
+// CHECK:STDERR: fail_todo_import_api.carbon: error: semantics TODO: `Unsupported: Importing C++ functions that require thunks indirectly` [SemanticsTodo]
+// CHECK:STDERR:
+
+library "[[@TEST_NAME]]";
+
+import library "api";
+
+fn F() {
+  //@dump-sem-ir-begin
+  FooShort(8 as i16);
+  FooInt(88);
```

Comment:
The number 88 has some problematic associations in some cultures; let's pick a different arbitrary number here.